app/backend/requirements.txt

```diff
@@ -1,4 +1,4 @@
-azure-identity==1.12.0
+azure-identity==1.13.0b3
```

Comment:
Why do we need a dependency of a preview version of `azure-identity`?

Reply:
Because it has AzureDeveloperCredential, which is needed for this to work locally and remotely.